src/BigCommerce/Api/Carts/CartsApi.php

```diff
@@ -33,22 +33,53 @@ class CartsApi extends UuidResourceApi
     private const CARTS_ENDPOINT = 'carts';
     private const CART_ENDPOINT  = 'carts/%s';
 
-    public function get(): CartResponse
+    /**
+     *  The Cart returns an abbreviated result. Use this to return physical items product options.
+     */
+    public const INCLUDE_PHYSICAL_ITEMS = 'line_items.physical_items.options';
+
+    /**
+     * The Cart returns an abbreviated result. Use this to return digital items product options.
+     */
+    public const INCLUDE_DIGITAL_ITEMS = 'line_items.digital_items.options';
+
+    public function get(?string $include = null): CartResponse
     {
-        return new CartResponse($this->getResource());
+        $query = $include ? ['include' => $include] : [];
+        $response = $this->getClient()->getRestClient()->get(
```

Comment:
This manual access to the rest client can be replaced with `$this->getResource($query);`